src/macports1.0/reclaim.tcl

```diff
@@ -116,7 +117,7 @@ namespace eval reclaim {
 
         set retval 0
         if {[info exists macports::ui_options(questions_yesno)]} {
-            set retval [$macports::ui_options(questions_yesno) "" "" "" "y" 0 "Would you like to delete the entire build directory (including the default ccache location)?"]
+            set retval [$macports::ui_options(questions_yesno) "" "" "" "y" 0 "Would you like to delete everything under the build directory (except the default ccache location)?"]
```

Comment:
"everything under the build directory" might be better phrased as "the contents of the build directory" or "all port build directories". The latter could remove the need to specify that the ccache dir is excepted.